includes/class-sensei-lesson.php

```diff
@@ -285,7 +285,6 @@ public function lesson_prerequisite_meta_box_content() {
 			'order'            => 'ASC',
 			'exclude'          => $post->ID,
 			'suppress_filters' => 0,
-			'post_status'      => [ 'publish', 'draft' ],
```

Comment:
I think we'll need this in order for the prerequisite to show up in the dropdown for the duplicated lesson, won't we? In my earlier comment, I just meant removing the "- Draft" text from the end of the lesson name in the dropdown, which you've done below. 🙂

Reply:
Oh! I'm sorry for the confusion! Adding this part again =)

Reply:
To make reviewing easier, when you resolve these conversations would you mind dropping a link to the commit where you fixed it? Otherwise reviewers need to hunt through the changes to find the one they're looking for. Thx!

Reply:
Sorry! I didn't add to other changes too, but I'll add on the nexts ;)